packages/react-component-library/src/components/AutocompleteE/hooks/useAutocomplete.ts

```diff
@@ -32,7 +32,7 @@ export function useAutocomplete(
           if (!React.isValidElement(item)) {
             return false
           }
-          const filter = inputValue.toLowerCase()
+          const filter = (inputValue as string).toLowerCase()
```

Comment:
This looks like a mistake in the types based on the examples in the Downshift docs, so I went for a type assertion here.